src/scripts/modal.js

```diff
@@ -0,0 +1,27 @@
+console.log(`object`);
```

Comment:
Don't commit console logs. You can make a partial commit if needed. Like this:
```bash
git commit -p src/scripts/modal.js
```

Reply:
This console log is unnecessary, I removed it.

Reply:
Good 👍